### DISCUSSION TITLE
Should we just keep blank lines between sequence diagrams?

### DISCUSSION THREAD

**Discussion by cvchaparro** [DevOps-MBSE/AaC#149] [Archive] [answered]
In the `puml_sequence` function there was a `TODO` comment saying we should revisit putting blank lines between sequence diagrams. So, should we keep blank lines between sequence diagrams, or do use else between them?

I'm removing the `TODO` comment to fix linter warnings so we need to track the question here.

**Comment by Coffee2Bits** [comment]
I don't think we should remove it, the space is a good delimiter between multiple diagrams.

**Comment by jondavid-black** [comment, marked as answer]
I agree with keeping the space and removing the TODO.

**Comment by lizzcondrey** [comment]
This has been marked resolved, has an answer, and based on review of the `plant_uml` plugin the blank line and no TODO persist. Closing this discussion.
